ql/src/java/org/apache/hadoop/hive/ql/exec/FetchOperator.java

```diff
@@ -215,7 +216,7 @@ public void setWork(FetchWork work) {
 
   @SuppressWarnings("unchecked")
   public static InputFormat getInputFormatFromCache(
-    Class<? extends InputFormat> inputFormatClass, Configuration conf) throws IOException {
+      Class<? extends InputFormat> inputFormatClass, Configuration conf) throws IOException {
```

Comment:
unnecessary ws change.

Reply:
@ashutoshc This was a checkstyle violation which is fixed by this patch.